nucypher/cli/painting/status.py

```diff
@@ -59,17 +40,9 @@ def paint_contract_status(registry, emitter):
 Registry ................. {registry.filepath}
     """
 
-    confirmed, pending, inactive = staking_agent.partition_stakers_by_activity()
```

Comment:
maybe not in this PR (maybe in this) we can still gather info about staking providers, like active/not active

Reply:
Absolutely, status needs a revival.